bunkering/BunkeringMDP.py

```diff
@@ -0,0 +1,174 @@
+from __future__ import annotations
+
+import math
+from enum import Enum
+from mcts4py.MDP import *
+import random
+import numpy as np
+
+KAPPA=10
+KAPPA2 = 100
+
+class PortEnums(Enum):
```

Comment:
Probably no need to make it this complicated. A simple Python dictionary will suffice.